src/components/Application-Body/header/Header.tsx

```diff
@@ -15,24 +17,27 @@ import PostNewJob from './PostNewJob/PostNewJob';
 
 import { getAllSelectOptions } from '../../../server/ads';
 import { AllSelectOptions } from '../../../types/AllSelectOptions';
+import { UserStore } from '../../../store/UserStore';
 
 interface HeaderOwnProps {
     fetchAllAdsAfterPost: () => void;
     searchValue: string;
     onSearchValueChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    userStore?: UserStore;
 }
 
 const Header: React.FC<HeaderOwnProps> = (props): JSX.Element => {
     const { fetchAllAdsAfterPost, onSearchValueChange, searchValue } = props;
     const classes = styles({});
+    const userStore: UserStore = props.userStore!;
 
     const [allSelectOptions, setAllSelectOptions] = useState<AllSelectOptions | null>(null);
     const [openAddDialog, setOpenAddDialog] = useState<boolean>(false);
     const addButtonRef = useRef(null);
 
     // when closing dialog - blur the add button
     useEffect(() => {
-        if (!openAddDialog && addButtonRef) {
+        if (userStore.isRamad && !openAddDialog && addButtonRef) {
```

Comment:
Ohhh no.... we have MR's with conflicts here 😓 😅  (Since I moved some sections of the code)